style_about.css

```diff
@@ -0,0 +1,751 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
- [ ] For this project, part of the design requirement was to use the fonts `Cocogoose` and `Lato.` Kindly use these fonts for this project. 👍